plugins/modules/proxmox_storage_contents_info.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright Julian Vanden Broeck (@l00ptr) <julian.vandenbroeck at dalibo.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: proxmox_storage_contents_info
+short_description: List content from a Proxmox VE storage
+version_added: 8.2.0
+description:
+  - Retrieves information about stored objects on a specific storage attached to a node.
+options:
+  storage:
+    description:
+      - Only return content stored on that specific storage.
+    aliases: ['name']
+    type: str
+    required: true
+  node:
+    description:
+      - Proxmox node to which the storage is attached.
+    type: str
+    required: true
+  content:
+    description:
+      - Filter on a specific content type.
+    type: str
+    choices: ["all", "backup", "rootdir", "images", "iso"]
+    default: "all"
+  vmid:
+    description:
+      - Filter on a specific VMID.
+    type: int
+author: Julian Vanden Broeck (@l00ptr)
+extends_documentation_fragment:
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+"""
+
+
+EXAMPLES = """
+- name: List existing storages
+  community.general.proxmox_storage_contents_info:
+    api_host: helldorado
+    api_user: root@pam
+    api_password: "{{ password | default(omit) }}"
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+    storage: lvm2
+    content: backup
+    vmid: 130
+"""
+
+
+RETURN = """
+proxmox_storage_content:
+  description: Content of of storage attached to a node.
+  type: list
+  returned: success
+  elements: dict
+  contains:
+    content:
+      description: Proxmox content of listed objects on this storage
+      type: str
+      returned: success
+    ctime:
+      description: Creation time of the listed objects
+      type: str
+      returned: success
+    format:
+      description: Format of the listed objects (can be raw, pbs-vm, iso,...)
+      type: str
+      returned: success
+    size:
+      description: Size of the listed objects
+      type: int
+      returned: success
+    subtype:
+      description: Subtype of the listed objects (can be qemu or LXC)
```

Comment:
Is the string literally `qemu` or `LXC`?
```suggestion
      description: Subtype of the listed objects (can be V(qemu) or V(LXC)).
```
Or is it lower-case `lxc`?
```suggestion
      description: Subtype of the listed objects (can be V(qemu) or V(lxc)).
```

Reply:
lowercase lxc and not LXC.